content/source/docs/enterprise/admin/backup-restore.html.md

```diff
@@ -5,35 +5,56 @@ page_title: "Backups and Restores - Infrastructure Administration - Terraform En
 
-Backing up and restoring into a new installation of Terraform Enterprise is the only supported way to migrate between deployment types (Standalone vs. Clustered) and operational modes (*External Services* vs. *Mounted Disk*, etc.).
+Using the backup and restore API is the only supported way to:
 
-## About Backups
+- Migrate between deployment types (Standalone, Clustered).
+- Migrate between operational modes (Mounted Disk, External Services).
 
-Terraform Enterprise's backup utility backs up all of the data stored in a Terraform Enterprise installation, including both the blob storage and the PostgreSQL database. It does not back up the the installation configuration.
+## About Backups and Restores
 
-See [Data Security](../system-overview/data-security.html) for details about the contents of Terraform Enterprise's blob storage and database.
+The backup and restore API backs up all of the data stored in a Terraform Enterprise installation, including both the blob storage and the PostgreSQL database. It does not back up the installation configuration. This backup can then be restored to a new installation of Terraform Enterprise.
 
-### API Authentication Token
+Please note the following when using the backup and restore API:
 
-Calls to the backup API must be authenticated with a special bearer token, which can be found on the settings dashboard (`https://<TFE HOSTNAME>:8800/settings`) near the bottom of the page:
+- The version of Terraform Enterprise cannot be changed between a backup and restore. That is, a backup taken from one version of Terraform Enterprise cannot be restored to an installation running a different version of Terraform Enterprise.
+- The Terraform Enterprise installation that will be restored to must be a new, running installation with no existing application data.
+- Once a restore is completed, the Terraform Enterprise application will need to be restarted before it can use the restored data.
+
+See also:
+
+- [Data Security](../system-overview/data-security.html) for details about the contents of Terraform Enterprise's blob storage and PostgreSQL database.
+
+### Authentication 
+
+The backup and restore API uses a separate authorization token which can be found on the settings dashboard (`https://<TFE HOSTNAME>:8800/settings`) near the bottom of the page:
 
 ![Screenshot: the TFE install dashboard, with the API token visible](./images/token.png)
 
-~> **Important:** Since this token can access all of the data in a Terraform Enterprise installation, protect it very carefully.
+-> **Note:** This authorization token is specific to the Terraform Enterprise installation. Unless otherwise specified during installation of the new Terraform Enteprise instance, the authorization token used to create a backup will be different than the authorization token used to perform a restore.
```

Comment:
Open to suggestions on better wording for this section. Specifically this note.

Reply:
Maybe this?

```suggestion
-> **Note:** This authorization token is specific to the Terraform Enterprise installation. If restoring to a separate Terraform Enterprise installation, the authorization token will be different for restore than it was for backup. Please ensure you are using the correct authorization token for the Terraform Enterprise installation you wish to backup from or restore to.
```

Reply:
> **Note:** This authorization token is specific to the Terraform Enterprise installation. For example, the authorization token will be different when restoring to a separate Terraform Enterprise installation. Please ensure you are using the correct authorization token of the instance you are communicating with when performing a backup or restore operation.

Reply:
Submitted changes!